schemas/ontouml2.schema.json

```diff
@@ -88,6 +88,44 @@
         },
         "isDerived": {
           "$ref": "#/definitions/isDerived"
+        },
+        "isExtensional": {
+          "description": "A boolean field that captures if a class decorated as «collective» is extensionally defined (i.e., it's parts cannot change).",
+          "$ref": "#/definitions/nullableBoolean"
+        },
+        "isPowertype": {
+          "description": "A boolean field that captures if a class decorated as «type» represents a powertype.",
+          "$ref": "#/definitions/nullableBoolean"
+        },
+        "order": {
+          "description": "A string field that captures the type-order of a class decorated as «type».",
+          "$ref": "#/definitions/nullableString"
+        },
+        "allowed": {
+          "description": "A nullable array of unique enumerated strings that represents the allowed (possible) ontological natures of the class's instances.",
+          "oneOf": [
+            {
+              "type": "null"
+            },
+            {
+              "type": "array",
+              "uniqueItems": true,
+              "minItems": 1,
+              "items": {
+                "type": "string",
+                "enum": [
+                  "object",
```

Comment:
Didn't we agree to call this nature functional-complex? Object in UFO encompasses functional complexes, collectives and quantities.